modules/stochastic_tools/src/utils/PolynomialQuadrature.C

```diff
@@ -145,6 +153,18 @@ Legendre::quadrature(const unsigned int order,
   gauss_legendre(order, points, weights, _lower_bound, _upper_bound);
 }
 
+void
+Legendre::clenshawQuadrature(const unsigned int order,
```

Comment:
It seems like this calculation should be in the Quadrature object, why is it needed in the Polynomial object?

Reply:
Different polynomials will have different Clenshaw quadratures. It is all based on the weighting function used. Legendre is easy since the weighting function is 1, which is why I decided to start with that one. Hermite is a bit more complicated, I actually have not seen Clenshaw-Hermite in any literature, which is why I didn't want to spend too much time on implementation and verification before it was deemed useful. But I still want to keep the functionality for possible future need.